addons/website_forum/views/website_pages_views.xml

```diff
@@ -110,7 +110,7 @@
     <field name="model">forum.post</field>
     <field name="priority">99</field>
     <field name="arch" type="xml">
-        <tree create="false" type="object" action="go_to_website">
+        <tree js_class="website_pages_list" create="false" type="object" action="go_to_website" multi_edit="1">
```

Comment:
Actually, I tested in master: not using that class may have been wanted... or be good by chance. Before your change with the SearchModel, we do not want any of its behavior here. Actually already in the cases we already use it it seems broken... damn... :open_mouth: 

Master:
- Go to Site -> Products
- Select some products
- Actions -> Delete
=> The page delete dialog opens... and crash if you try to use it :sweat_smile: 

Probably to check with all model and all behaviors linked to this website_pages_list? :grimacing: